Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityArticulatedHand.cs

```diff
@@ -194,19 +207,19 @@ private async void SetHandMeshObserver(SpatialInteractionSourceState sourceState
         {
             handMeshObserver = await sourceState.Source.TryCreateHandMeshObserverAsync();
         }
-#endif
+#endif // WINDOWS_UWP
 
         /// <summary>
         /// Update the hand data from the device.
         /// </summary>
         /// <param name="interactionSourceState">The InteractionSourceState retrieved from the platform.</param>
         private void UpdateHandData(InteractionSourceState interactionSourceState)
         {
-#if WINDOWS_UWP
+#if DOTNETWINRT_PRESENT || WINDOWS_UWP
```

Comment:
same as earlier nit comment...

Reply:
Updated to `WINDOWS_UWP || DOTNETWINRT_PRESENT`